skimage/feature/_fisher_vector.py

```diff
@@ -79,8 +79,9 @@ def learn_gmm(descriptors, *, n_modes=32, gm_args=None):
 
     Examples
     --------
-    >>> import pytest
-    >>> _ = pytest.importorskip('sklearn')
+    .. testsetup::
+        >>> import pytest; _ = pytest.importorskip('sklearn')
+
```

Comment:
doctestplus has its own mechanism of [skipping doctests](https://github.com/scientific-python/pytest-doctestplus?tab=readme-ov-file#doctest-dependencies) if dependencies are missing. However, it seems that using `__doctest_skip__` will not show that tests were skipped and I couldn't get `.. doctest-requires::` to work. So instead I'm using an approach based on `.. testsetup` and `pytest.importorskip`.